### PR TITLE
KIALI-2927 combine overview and features

### PR DESCRIPTION
**Pull request by jshaughn**
There is a lot of redundancy in the Features and Overview pages.  Overview has a mixture of new and dated material.  Simplify by merging Overview into Features.

Make features bookmarkable and add TOC.

### CODE REVIEW

File: content/documentation/features/index.adoc [hunk 1]
```diff
@@ -0,0 +1,421 @@
...
+
+==== Graph: Traffic Animation
+Kiali offers several display options for the graph, including traffic animation.
+
+For HTTP traffic, circles represent successful requests while red diamonds represent errors. The more dense the circles and diamonds the higher the **request rate**. The faster the animation the faster the **response times**.
+
+TCP traffic is represented by offset circles where the speed of the circles indicates the traffic speed.
+++++
+<a class="video-popup" href="/images/documentation/features/kiali_traffic_animation.mp4" title="Visualize your traffic flow">
+    <video autoplay muted loop width="600px" src="/images/documentation/features/kiali_traffic_animation_thumb.mp4"/>
```
Comment (josejulio) [inline content/documentation/features/index.adoc]: The same for videos:

```html
<video style="display:block;margin:0 auto;" autoplay muted loop width="600px" src="/images/documentation/features/kiali_traffic_animation_thumb.mp4"/>
```

Comment (jshaughn) [reply content/documentation/features/index.adoc]: OK, will do videos as well.

File: content/documentation/features/index.adoc [hunk 2]
```diff
@@ -0,0 +1,421 @@
...
+
+=== Graph
+The graph provides a powerful way to visualize the topology of your service mesh.  It shows you which services communicate with each other and the **traffic rates** and **latencies** between them.  It lets you visually identify problem areas ane quickly pinpoint issues.  Kiali provides **four different graph types** giving you a high-level view of service interactions, a low level view of pods, or a logical view of applications.
+
+The graph also lets you see which services are **configured** with such things as virtual services and circuit breakers. It also identifies **security** issues by identifying traffic not configured as expected.  And you can observe the **traffic flow** between components by watching the animation or viewing the **metrics**.
+
+You can configure the graph to show the namespaces and data that is important to you, and display it in the way that best meets your needs.
+++++
+<a class="image-popup-fit-height" href="/images/documentation/features/graph-overview.png" title="Visualize your service mesh topology">
+    <img src="/images/documentation/features/graph-overview-thumb.png"/>
```
[the remark this answers is not in the document]
Comment (jshaughn) [reply content/documentation/features/index.adoc]: OK, will do.

**Comment by jshaughn** [comment]
@josejulio I updated the images to be centered, it looks better, thanks,

> About the content, some info in the validations table show as deprecated, maybe we should put a reason of the deprecation and what is done instead?

I pinged @burmanm to review the validations content.  If he gets back to me soon I can make updates, but we could also defer that to a separate PR.

**Comment by josejulio** [comment]
> I pinged @burmanm to review the validations content. If he gets back to me soon I can make updates, but we could also defer that to a separate PR.

A separete PR is OK, just wanted to bring that up

**Comment by jshaughn** [comment]
@josejulio @burmanm  said there are several changes to the validation information and he will take it as a separate task after this work is in.

Also, that link discovery tool is great.  I already had the /features/ alias in place but my alias for /documentation/overview/ was wrong.  I have fixed it.

**Comment by mwringe** [comment]
A quick glance and this looks go to me. Glad to see we don't have as much duplication as before

**Pull request merged**